TAScheduler/AcceptanceTests/InstructorToCourse tests.py

```diff
@@ -0,0 +1,149 @@
+from django.test import TestCase
+
+from TAScheduler.models import User, Instructor, Course, InstructorToCourse
+
+
+class SuccessfulCreation(TestCase):
+    user = None
+    instructor = None
+    course = None
+    instrToCourse = None
+
+    def setUp(self):
+        self.user = User.objects.create(email_address="instructor@class.com", password="password", first_name="first",
+                                        last_name="last", home_address="there", phone_number=1234567890)
+
+        self.instructor = Instructor.objects.create(user=self.user)
+
+        self.course = Course.objects.create(course_id=100, semester="fall 2023", name="test",
+                                            description="test", num_of_sections=2, modality="online", credits=1)
+
+        self.instrToCourse = InstructorToCourse.objects.create(instructor=self.instructor, course=self.course)
+
+    # /home/managecourse/addinstructor
+    def test_creation(self):
+        temp = InstructorToCourse.objects.create(instructor=self.instructor, course=self.course)
+        self.assertIsNotNone(temp, "Did not make InstructorToCourse")
+        self.assertEqual(temp.instructor, self.instructor, "Instructor is not correct instructor")
+        self.assertEqual(temp.course, self.course, "Course is not correct course")
+
```

Comment:
Again, you have the correct logic but need the response object mentioned previously. Once simulated a .post() using your temp variables, do the similar checks you are going here: do a query in the Database for the instruct, course, and instructor To Course to make sure that the assignment "actually created".

Also, add another few tests to check to check: instructor's course = temp course ... course's instructor = temp instructor ... instructorToCourse = both temp instructor & temp course ....
to ensure that they are assigned to each other

Need test for confirmation message resp.context["message"] = "successfully assigned instructor -> course"